docs/spark/spark-writes.md

```diff
@@ -311,7 +311,9 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
+To write with SQL, both global sort (`ORDER BY`) and local sort (`SORT BY`) work for the requirement.
```

Comment:
"the requirement" was always a bit confusing to me here. What do you think about my suggestions in this section?

Reply:
yes i think so too.  The revised sentence is better.